Lab1_Operationalizing_Pytorch_with_Mlflow/src/bert-classifier/torchserve/Dockerfile

```diff
@@ -9,25 +9,20 @@ ENV MLFLOW_TRACKING_URI=http://35.185.111.8:5000
 ENV GCP_CREDS_JSON_BASE64_ENV=${GCP_CREDS_JSON_BASE64}
 ENV GOOGLE_APPLICATION_CREDENTIALS=${MLFLOW_HOME}/gcp.json
 
-COPY requirements.txt /tmp/
 RUN mkdir -p ${MLFLOW_HOME}/scripts && \
     mkdir -p ${MODEL_STORE} && \
     echo "${GCP_CREDS_JSON_BASE64_ENV}"| base64 --decode > "${GOOGLE_APPLICATION_CREDENTIALS}" && \
-    mkdir -p /usr/share/man/man1/ && \
-    apt-get update && apt-get install --no-install-recommends -y git default-jdk wget && \
-    apt-get clean -y -qq && apt-get autoremove -y -qq && \
-    rm -rf /var/lib/apt/lists/* && \
     git clone https://github.com/pytorch/serve.git && \
     cd serve && python ./ts_scripts/install_dependencies.py && \
```

Comment:
This `install_dependencies.py` script actually installs the correct version of pytorch for your system, e.g. if you have CUDA or even *gasp* windows!??  

I think it makes sense in the base image, and works best run before the `pip install requirements.txt`

Reply:
So you're suggesting we put TorchServe in both Trainer and Serve images?

Reply:
Yep, well at least clone the repo and install the dependencies of torchserve.
Probably just makes sense to do the whole lot then I suppose?

Reply:
Sorry, ignore the whole lot part

Reply:
Ok, I'll rebuild the base image and update the PR

Reply:
@savagej done